### ISSUE TITLE
Crash as tapping on "Open Source" from settings

### ISSUE THREAD

**Issue by mizzunet** [appstome/timetome-app#13]
```
FATAL EXCEPTION: main
Process: me.timeto.app, PID: 23749
android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
	at android.app.ContextImpl.startActivity(ContextImpl.java:1127)
	at android.app.ContextImpl.startActivity(ContextImpl.java:1103)
	at android.content.ContextWrapper.startActivity(ContextWrapper.java:436)
	at l7.u.h(Unknown Source:22)
	at m7.i0.a(Unknown Source:47)
	at m7.i0.q(Unknown Source:110)
	at x.k.a(Unknown Source:40)
	at x.k.q(Unknown Source:42)
	at m.n.p0(Unknown Source:419)
	at p.h2.h(Unknown Source:133)
	at e6.a.t(Unknown Source:8)
	at u5.k.T0(Unknown Source:80)
	at u6.g.n(Unknown Source:137)
	at u6.g.A(Unknown Source:44)
	at u6.g.t(Unknown Source:16)
	at e1.s0.J0(Unknown Source:50)
	at e1.s0.G(Unknown Source:30)
	at o.d.G(Unknown Source:8)
	at e1.h.e(Unknown Source:180)
	at e1.h.e(Unknown Source:159)
	at e1.h.e(Unknown Source:159)
	at e1.h.e(Unknown Source:159)
	at e1.h.e(Unknown Source:159)
	at e1.h.e(Unknown Source:159)
	at e1.i.e(Unknown Source:25)
	at i.x.d(Unknown Source:28)
	at d0.y.d(Unknown Source:195)
	at androidx.compose.ui.platform.AndroidComposeView.D(Unknown Source:78)
	at androidx.compose.ui.platform.AndroidComposeView.i(Unknown Source:223)
	at androidx.compose.ui.platform.AndroidComposeView.dispatchTouchEvent(Unknown Source:87)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3123)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2804)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3123)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2804)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3123)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2804)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3123)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2804)
	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:490)
	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1907)
	at android.app.Activity.dispatchTouchEvent(Activity.java:4412)
	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:448)
	at android.view.View.dispatchPointerEvent(View.java:15928)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:7246)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:7032)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6435)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6492)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6458)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6623)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6466)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6680)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6439)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6492)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6458)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6466)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6439)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:9435)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:9386)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:9355)
	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:9561)
	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:267)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:335)
	at android.os.Looper.loopOnce(Looper.java:162)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8282)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:554)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978)
	Suppressed: z6.e: [androidx.compose.ui.platform.v1@dc4bc62, e0.e@3f641f3, n1{Cancelling}@ce44eb0, u0@94f5129]
```

Thanks a lot for coming up such a nice app :)

Pixel 4a
Evolution X 14

**Comment by appstome** [comment]
Oops, I broke the most important button :)

Fixed in 2024.02.24.483. Already on Google Play, F-Droid apk is here https://github.com/appstome/timetome-app/releases/tag/v2024.02.24.483, soon on F-Droid itself.

Thanks for the bug-report.

**Comment by appstome** [comment]
Starting today on F-Droid.
